admin-dev/themes/new-theme/scss/components/_submittable_input.scss

```diff
@@ -0,0 +1,33 @@
+.ps-submittable-input-wrapper {
+  position: relative;
+
+  .check-button {
+    position: absolute;
+    top: 0;
+    right: 1px;
+    z-index: 3;
+    width: 33px;
+    height: 100%;
+    padding: 0;
+    cursor: pointer;
+    background: #bbcdd2;
```

Comment:
I see other usages of this core in the new-theme. Doesn't it deserve a variable ?
@NeOMakinG

Reply:
Damn, I even think we already have a variable inside the UIKit, that could be something to improve, missed it :(

Reply:
no problem, we can still fix it. It is merged, but open for improvements